src/ViewRendererBundle/objects/helper/TPkgViewRendererSnippetDirectory.class.php

```diff
@@ -408,6 +408,12 @@ public function getBasePaths($oPortal = null, $sBaseDirectory = null)
             }
         }
 
+        if (true === $this->requestInfoService->isBackendMode()) {
+            $aBasePaths[$sPortalCacheId] = $this->addBundlePaths($aBasePaths[$sPortalCacheId], $sBaseDirectory);
+
+            // TODO or also use a "snippet chain" field for the backend theme? (table cms_config_themes)
```

Comment:
_Discuss here_

Reply:
We could use themes if we assigned a theme via cms_config. 

That solution would make the handling consistent with our frontend handling - making it easier to understand. At the same time it makes it much easier to add/overwrite twig files for the backend.

Reply:
Ok.

(Frontend) themeing used here.